ui/apps/platform/src/Containers/Vulnerabilities/components/HeaderLoadingSkeleton.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Flex, Skeleton } from '@patternfly/react-core';
+
+export type HeaderLoadingSkeletonProps = {
+    nameScreenreaderText: string;
+    metadataScreenreaderText: string;
+};
+
+/**
+ *  Layout component for loading state of a page header.
+ * @param {string} nameScreenreaderText - Screenreader text for the name skeleton.
+ * @param {string} metadataScreenreaderText - Screenreader text for the metadata skeleton.
+ * @returns {JSX.Element}
+ */
```

Comment:
nit: The JSDoc seems a little redundant, considering the types and the well-chosen variable names.